internal/cmdinit/cmdinit_test.go

```diff
@@ -160,10 +160,15 @@ func TestCmd_failNotExists(t *testing.T) {
 	d, err := ioutil.TempDir("", "kpt")
 	assert.NoError(t, err)
 
-	r := cmdinit.NewRunner("kpt")
+	out := &bytes.Buffer{}
+	log := &bytes.Buffer{}
+
+	r := cmdinit.NewRunner(fake.CtxWithFakePrinter(nil, nil), "kpt")
 	r.Command.SetArgs([]string{filepath.Join(d, "my-pkg"), "--description", "my description"})
 	err = r.Command.Execute()
 	if assert.Error(t, err) {
 		assert.Contains(t, err.Error(), "does not exist")
 	}
+	assert.Equal(t, "", out.String())
+	assert.Equal(t, "", log.String())
```

Comment:
What is this testing? It seems like the buffers aren't being used, so they can never contain anything else than `""`.

Reply:
Removed.